### PR TITLE
[CARE-1075] Unify vertical spacing of content

### PR DESCRIPTION
**Pull request by oleg-semyonov**
- Moved all vertical spacing around the content from child blocks into the `Layout` component for consistency
- Cleaned up styles
- Add styles to remove bottom margin on the last block in the story.

### CODE REVIEW

File: components/SlateRenderer/SlateRenderer.module.scss [hunk 1]
```diff
@@ -10,6 +10,10 @@
         /* stylelint-disable selector-class-pattern, max-nesting-depth, selector-max-compound-selectors */
         .prezly-slate-document {
             color: inherit;
+
+            > *:last-child {
+                margin-bottom: 0 !important;
+            }
         }
```
Comment (alpower) [inline components/SlateRenderer/SlateRenderer.module.scss:17]: nice one - this is exactly how I hoped you would have done it 🥇

**Review by alpower — APPROVED** [review]

**Review by kudlajz — APPROVED** [review]

**Comment by amandolino** [comment]
When I check the 'prezly-slate-document' section in stories it seems have a bottom margin of minimum 40px that adds to the total 🤔

**Comment by oleg-semyonov** [comment]
> When I check the 'prezly-slate-document' section in stories it seems have a bottom margin of minimum 40px that adds to the total 🤔

That might just be an empty paragraph inserted in the editor. Nothing we can really do about it. Can you share an example URL for me to check?

**Comment by oleg-semyonov** [comment]
Yeah, it's an empty paragraph there. This should be cleaned up by the backend or editor (or the story author 😉)
image

**Pull request merged**